### PR TITLE
[TTAHUB-3416/3417] Fixes for something went wrong page

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change
- Fixes two bugs: Something went wrong page was missing a page title, and using the back button from that page got you mixed results with the side navigation (it would usually stay disappeared)
- Refactored to abandon the context idea, we now use a more traditional model of error pages, required a few changes to tests

## How to test
Confirm that the something went wrong page still functions as it originally did, with the exception of the page now having a meaningful title and the back button bug being resolved

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3416
* https://jira.acf.gov/browse/TTAHUB-3417

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Before merge to main

- [ ] OHS demo complete
- [ ] Ready to create production PR

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/Routes.js [hunk 1]
```diff
@@ -0,0 +1,420 @@
+import React from 'react';
```
Comment (AdamAdHocTeam) [inline frontend/src/Routes.js:1]: Nice clean up app.js

**Review by AdamAdHocTeam — COMMENTED** [review]
If I modify the code for an exception to pass a 500 status code to the something-went-wrong route it shows as a 400. Not sure if this is a bug or something I did wrong.

Otherwise looks good.

**Comment by thewatermethod** [comment]
> If I modify the code for an exception to pass a 500 status code to the something-went-wrong route it shows as a 400. Not sure if this is a bug or something I did wrong.

Not sure how to recreate this; I tested it with the view activity reports page and confirmed that the HTTP code in the network tab matched the place I was redirected to.

**Review by nvms — APPROVED** [review]
Looks and works great. I like the refactor of moving routes into their own component 👏

**Review by AdamAdHocTeam — APPROVED** [review]

**Pull request merged**